app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    key = os.getenv("API_KEY")
+    response = requests.get(
+        "http://api.weatherapi.com/v1/current.json",
+        params={"q": "Paris", "key": key},
+    )
```

Comment:
According to the checklist, you should use constants for the URL and the city name. This will make your code more maintainable. You can define these constants at the top of your file. In addition, the constant names should be written in uppercase. For example: `URL = 'http://api.weatherapi.com/v1/current.json'` and `CITY = 'Paris'`.